src/_content/posts/childcare-during-djangocon.md

```diff
@@ -0,0 +1,31 @@
+---
+author: Nathan Zeager
+category: General
+date: 2024-07-31 06:00:00
+title: "Childcare During DjangoCon US"
+
+cover:
+    url: /assets/img/blog/childcare-during-djangocon.jpg
+    alt: "Childcare During DjangoCon US"
+
+---
+
+Childcare at DjangoCon US and tech conferences more generally is an important factor that helps folks attend who otherwise may not be able to. We want to make sure that parents feel welcome bringing their children to DjangoCon US and that the availability of childcare is not a barrier to attending.
+
+To assist with that, DjangoCon US will reimburse you for childcare during the conference(up to $150 per day) with a receipt. To request a reimbursement, please email hello@defna.org.
```

Comment:
```suggestion
To assist with that, DjangoCon US will reimburse you for childcare during the conference(up to $150 per day) with a receipt.

For nursing parents, we will provide a clean, comfortable, and private lactation space including a small refrigerator to store pumped milk, labeling supplies, and antibacterial wipes.

Please contact us at hello@defna.org if you are interested in utilizing DjangoCon US’s child care reimbursement or lactation space services.
```

I don't think we've called out the lactation room yet. I think it's something we should market here and then determine if we need based on interested. This also adjusts the language to have them email before making arrangements to avoid a large number of unexpected reimbursements.